r55/src/exec.rs

```diff
@@ -418,9 +421,32 @@ fn dram_slice(emu: &mut Emulator, ret_offset: u64, ret_size: u64) -> Result<&mut
 }
 
 fn r55_gas_used(inst_count: &BTreeMap<String, u64>) -> u64 {
-    // Assuming 1 instruction = 1 gas rn
-    let num_inst = inst_count.iter().map(|(_, count)| count).sum::<u64>();
-    // This is the minimimum gas used to call a zero-logic r55 function with no arguments
-    let base_cost = 3_933_428;
-    num_inst - base_cost
+    let total_cost = inst_count
+        .iter()
+        .map(|(instr_name, count)|
```

Comment:
```suggestion
        .map(|(inst_name, count)|
```
How about unifying as `inst` since it's `inst_count`?